Source/Flow/Private/Nodes/Route/FlowNode_Start.cpp

```diff
@@ -13,11 +13,201 @@ UFlowNode_Start::UFlowNode_Start(const FObjectInitializer& ObjectInitializer)
 	bCanDelete = bCanDuplicate = false;
 #endif
 
-	InputPins = {};
-	AllowedSignalModes = {EFlowSignalMode::Enabled, EFlowSignalMode::Disabled};
+	OutputPins = { UFlowNode::DefaultOutputPin };
 }
 
 void UFlowNode_Start::ExecuteInput(const FName& PinName)
 {
 	TriggerFirstOutput(true);
 }
+
+void UFlowNode_Start::SetDataPinValueSupplier(IFlowDataPinValueSupplierInterface* DataPinValueSupplier)
+{
+	FlowDataPinValueSupplierInterface = Cast<UObject>(DataPinValueSupplier);
+}
+
+#if WITH_EDITOR
+
+bool UFlowNode_Start::TryAppendExternalInputPins(TArray<FFlowPin>& InOutPins) const
+{
+	// Add pins for all of the Flow DataPin Properties
+	for (const FFlowNamedDataPinOutputProperty& DataPinProperty : OutputProperties)
+	{
+		if (DataPinProperty.IsValid())
+		{
+			InOutPins.AddUnique(DataPinProperty.CreateFlowPin());
+		}
+	}
+	
+	return !OutputProperties.IsEmpty();
+}
+
+#endif // WITH_EDITOR
+
+// Must implement TrySupplyDataPinAs... for every EFlowPinType
+FLOW_ASSERT_ENUM_MAX(EFlowPinType, 12);
+
+FFlowDataPinResult_Bool UFlowNode_Start::TrySupplyDataPinAsBool_Implementation(const FName& PinName) const
```

Comment:
Start node needed a full suite of overrides to the supply functions, but this will generally not be necessary for most flow nodes.  Start node can have arbitrary pins added in the instance, so it needs to support all pin types and route them through its external value supplier.